lib/Modules/HyperVCIHooks/HyperVCIHooks.psm1

```diff
@@ -424,17 +375,17 @@ function Install-Projects ($buildFor) {
     
     # Individual projects
     # Nova
-    if ($buildFor -eq "openstack/nova") {
-        Invoke-BuildNova
+    if ($project -eq "openstack/nova") {
+        CopyNovaDefaultConfs
     }
     # Networking-Hyperv
-    if ($buildFor -eq "openstack/networking-hyperv") {
-        Invoke-BuildNeutron
+    if ($project -eq "openstack/networking-hyperv") {
+        CheckNeutronHypervBin
     }
 }
 
 
-function Invoke-BuildNova {
+function CopyNovaDefaultConfs {
     $novaBin = (Get-CharmServices)['nova']['binary']
```

Comment:
Shouldn't these 4 lines be in fact a function called "CheckNovaHypervBin"? (see the "CheckNeutronHypervBin" referenced a few lines above).